frontend/pages/g/_groupSlug/r/create/plaintext.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div>
+    <v-form ref="domUrlForm" @submit.prevent="createRecipe">
+      <div>
+        <v-card-title class="headline">{{ $t('recipe.create-recipe-from-text') }}</v-card-title>
+        <v-card-text>
+          <p>{{ $t('recipe.create-recipe-from-text-description') }}</p>
+          <v-container class="pa-0">
+            <v-row>
+              <v-col cols="12">
+                <v-textarea
+                  v-model="recipeText"
+                  :label="$t('recipe.enter-recipe-text')"
+                  :placeholder="$t('recipe.recipe-text-placeholder')"
+                  outlined
+                  auto-grow
+                  rows="6"
+                  :disabled="loading"
+                ></v-textarea>
+              </v-col>
+            </v-row>
+          </v-container>
+        </v-card-text>
+        <v-card-actions v-if="recipeText">
```

Comment:
Instead of hiding this, can we disable the create button? It's a bit jarring to see it pop in and out. Something like:
`<BaseButton :disabled="!recipeText" ... />